tests/suites/os/tests/connectivity/index.js

```diff
@@ -133,10 +147,25 @@ module.exports = {
 						await this.context
 							.get()
 							.worker.executeCommandInHostOS(
-								`ping -c 10 ${URL_TEST}`,
+								`curl ${URL_TEST}`,
 								this.context.get().link,
 							);
-						test.true(`${URL_TEST} responded over ${proxy} proxy`);
+
+						const proxyLog = await this.context
+							.get()
+							.worker.executeCommandInHostOS(
+								['balena',
+									'logs',
+									await getProxyContainerID(),
+									'|', 'tail', '-n1'].join(' '),
+								this.context.get().link,
+							);
+
+						test.match(
```

Comment:
```suggestion
            test.comment(`Looking for ${proxy} connection logs...`);
						test.match(
```
Adding some logs before commands that may fail.